test/cpp/api/imethod.cpp

```diff
@@ -8,38 +8,46 @@
 using namespace ::testing;
 using namespace caffe2;
 
-// TODO(T96218435): Enable the following tests in OSS.
+const char* simple = "torch/csrc/deploy/example/generated/simple";
```

Comment:
this path is OK?  I thought that maybe the reason the previous PR failed was that this path was not right given where it was run from.  Maybe it is OK though, if the test is run from the root...

Reply:
Probably okay. That's what the test_deploy.cpp uses. I bet the tests are run from the root.